salt/metalk8s/orchestrate/bootstrap/init.sls

```diff
@@ -69,6 +77,7 @@ Deploy CA role on bootstrap minion:
   - sls:
     - metalk8s.roles.ca
   - saltenv: metalk8s-{{ version }}
+  - pillar: {{ pillar_data | tojson }}
```

Comment:
Why?  The CA role should be fully stand-alone, no need to have any pillar values. Or am I missing something?

Reply:
It was failing when rendering `salt/metalk8s/repo/offline.sls`, the pillar `pillar.metalk8s.endpoints['package-repositories'].ip` was needed